go.mod

```diff
@@ -81,9 +81,9 @@ require (
 	gopkg.in/olivere/elastic.v3 v3.0.75
 	gopkg.in/olivere/elastic.v5 v5.0.84
 	gorm.io/driver/mysql v1.0.1
-	gorm.io/driver/postgres v1.0.0
+	gorm.io/driver/postgres v1.5.0
 	gorm.io/driver/sqlserver v1.0.4
-	gorm.io/gorm v1.20.6
+	gorm.io/gorm v1.24.7-0.20230306060331-85eaf9eeda11
```

Comment:
Will `gorm.io/gorm v1.24.7` not work? Surprised you need a specific commit like that

Reply:
😭 I'm not sure, but because so many things broke / changed I went back a version which sent it to `1.24.6` which is much cleaner